wic/iot2050-swu-secure.wks.in

```diff
@@ -15,6 +15,6 @@ part --source empty --align 1024 --fixed-size 4G --uuid "${ABROOTFS_PART_UUID_B}
 
 # persistent partitions
 part /var --source rootfs --rootfs-dir=${IMAGE_ROOTFS}/var --label var --fstype=ext4 --mkfs-extraopts "-T default" --align 1024  --size 512M
-part /home --source rootfs --rootfs-dir=${IMAGE_ROOTFS}/home --label home --fstype=ext4 --mkfs-extraopts "-T default" --align 1024  --size 512M
+part /home --source rootfs --rootfs-dir=${IMAGE_ROOTFS}/home --label home --fstype=ext4 --mkfs-extraopts "-T default" --align 1024  --size 1G
```

Comment:
Later maybe we can pass in a configuration variable so that user can specify the home size with kas menu.

Reply:
Yeah, it remains an example image. This here is to allow doing the example update with the example image.